contracts/shared/helpers/Fixture.ts

```diff
@@ -187,7 +189,14 @@ export default class Fixture {
   ) {
     await (
       await this.verificationGateway.processBundle(
-        this.bundleFrom(wallet, contract, method, params, nonce, ethValue),
+        await this.bundleFrom(
```

Comment:
After fixing `bundleFrom`, you shouldn't need `await` here.

Reply:
Addressed